### PR TITLE
Simplify boilerplate

### PR DESCRIPTION
**Pull request by fpagnoux**
Apply https://github.com/openfisca/openfisca-core/pull/467 to `france`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 14.1.1 - [#697](https://github.com/openfisca/openfisca-france/pull/697)
+
+* Amélioration technique
+* Utilise le module `core.formulas_toolbox` plutôt que de réimporter un par un tous les objets python nécessaire pour écrire une formule
```
Comment (fpagnoux) [inline CHANGELOG.md]: Rename to `module_api`

File: openfisca_france/model/base.py [hunk 2]
```diff
@@ -1,77 +1,8 @@
...
-from openfisca_core.base_functions import (
-    requested_period_added_value,
-    requested_period_default_value,
-    requested_period_last_or_next_value,
-    requested_period_last_value,
-    )
-from openfisca_core.formula_helpers import apply_thresholds, switch
-from openfisca_core.periods import MONTH, YEAR, ETERNITY
-
+from openfisca_core.formula_toolbox import *
```
Comment (fpagnoux) [inline openfisca_france/model/base.py]: We can replace the `import *` by `__all__.extend()`

Comment (fpagnoux) [inline openfisca_france/model/base.py]: But at the same time it's much less clear 🤔

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 14.1.1 - [#697](https://github.com/openfisca/openfisca-france/pull/697)
+
+* Amélioration technique
+* Utilise le module `core.module_api` plutôt que de réimporter un par un tous les objets python nécessaire pour écrire une formule
```
Comment (cbenz) [inline CHANGELOG.md]: Did you rename `model_api` to `module_api` willingly?

File: openfisca_france/model/base.py [hunk 4]
```diff
@@ -1,77 +1,8 @@
...
-from openfisca_core.base_functions import (
-    requested_period_added_value,
-    requested_period_default_value,
-    requested_period_last_or_next_value,
-    requested_period_last_value,
-    )
-from openfisca_core.formula_helpers import apply_thresholds, switch
-from openfisca_core.periods import MONTH, YEAR, ETERNITY
-
+from openfisca_core.module_api import *
```
Comment (cbenz) [inline openfisca_france/model/base.py]: Did you rename `model_api` to `module_api` willingly?

Comment (fpagnoux) [inline openfisca_france/model/base.py]: No :(
Good catch !

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 14.1.1 - [#697](https://github.com/openfisca/openfisca-france/pull/697)
+
+* Amélioration technique
+* Utilise le module `core.model_api` plutôt que de réimporter un par un tous les objets python nécessaire pour écrire une formule
```
Comment (cbenz) [inline CHANGELOG.md]: Typos :
- python -> Python
- nécessaire -> nécessaires

**Comment by fpagnoux** [comment]
5 fixups pour 2 commits, donc un de changelog. Ne plus ouvrir de PR le soir avant de partir ;)

**Pull request merged**